### PR TITLE
Dual rebranded into Aqua. Edited to show this change

### PR DESCRIPTION
**Pull request by DefectivenessYT**
image

Dual has rebranded into aqua - I've edited the original presence to show this change, and also put my name down as the contributor, as i will be continuing development on this presence.

### CODE REVIEW

**Comment by ririxidev** [comment]
ok wtf you are doing, dual didn't rebranded to aqua, I don't see anything like that. I go to dual.pw i still get dual, not aqua.

**Comment by DefectivenessYT** [comment]
updated https://dual.pw to redirect to Aqua to show this change
image
Above is also proof of my ownership of the Dual domain.

File: websites/A/Aqua/dist/metadata.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+  "version": "1.0.0",
+  "logo": "https://cdn.itsaqua.net/logo.png",
+  "thumbnail": "https://i.imgur.com/CiE0uNv.png",
+  "color": "#2980b9",
   "tags": [
     "radio",
-    "dualfm",
-    "dual",
+    "aqua",
+    "aqua radio",
```
Comment (ririxidev) [inline websites/A/Aqua/dist/metadata.json:18]: no spaces in tags

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/Aqua/dist/metadata.json (hunk 1 above)
Comment (DefectivenessYT) [reply websites/A/Aqua/dist/metadata.json]: fixed

**Comment by DefectivenessYT** [comment]
Any update @ririxidev?

**Review by ririxidev — APPROVED** [review]

File: websites/A/Aqua/dist/metadata.json [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+    "en": "Aqua radio brings you the greatest hits from around the world"
   },
-  "url": "dual.pw",
-  "version": "1.0.7",
-  "logo": "https://i.imgur.com/JQ1t9N5.png",
-  "thumbnail": "https://i.imgur.com/zIFABSa.png",
-  "color": "#D08CCF",
+  "url": "itsaqua.net",
+  "version": "1.0.0",
+  "logo": "https://cdn.itsaqua.net/logo.png",
```
Comment (ririxidev) [inline websites/A/Aqua/dist/metadata.json:12]: bruh didnt notice, upload logo to imgur

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/Aqua/dist/metadata.json [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
-  "url": "dual.pw",
-  "version": "1.0.7",
-  "logo": "https://i.imgur.com/JQ1t9N5.png",
-  "thumbnail": "https://i.imgur.com/zIFABSa.png",
-  "color": "#D08CCF",
+  "url": "itsaqua.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/dBqeZ3q.png",
+  "thumbnail": "https://i.imgur.com/CiE0uNv.png",
+  "color": "#2980b9",
```
Comment (Bas950) [inline websites/A/Aqua/dist/metadata.json:14]: why is that the thumbnail?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Aqua/dist/metadata.json (hunk 3 above)
Comment (DefectivenessYT) [reply websites/A/Aqua/dist/metadata.json:14]: was under the impression the thumbnail is the background on the site? like the little background

Comment (Bas950) [reply websites/A/Aqua/dist/metadata.json:14]: a promotional banner, or a screenshot of the website homepage.

Comment (DefectivenessYT) [reply websites/A/Aqua/dist/metadata.json:14]: ahhh okay - this is a super blurred screenshot of one of our pages, would you like me to submit it unblurred?

Comment (Bas950) [reply websites/A/Aqua/dist/metadata.json:14]: Yes, it only needs blurring if its NSFW

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**